### ISSUE TITLE
Any document avaliable?

### ISSUE THREAD

**Issue by MSKTHU** [NVIDIAGameWorks/donut#1]
I found it is very hard for me to read the source code for using this engine. I referred to the donut example but can still hardly get the point what purposes those classes and functions are designed for. So do you have any document or website describing the APIs?

**Comment by apanteleev** [comment]
Not really, no. This framework has been made open source mostly for our convenience to be referenced from the SDK samples. The only reasonably well-documented part of it is [NVRHI](https://github.com/NVIDIAGameWorks/nvrhi).
